# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xmlada/hotfix-gnat-maps.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xmlada-26.0.0/gprbuild/gpr/src/gpr-compilation-process.adb.vanilla	2026-10-06 19:38:48.247698457 +0300
+++ xmlada-26.0.0/gprbuild/gpr/src/gpr-compilation-process.adb	2026-10-06 19:39:02.435734550 +0300
@@ -41,7 +41,7 @@
    --  A set of key=value
 
    package Prj_Maps is new Containers.Indefinite_Ordered_Maps
-     (String, Env_Maps.Map, Env_Maps."<", Env_Maps."=");
+     (String, Env_Maps.Map, "<", Env_Maps."=");
    --  A set of project+language=map
 
    function "<" (Left, Right : Id) return Boolean is
